contrib/collectd/template.py

```diff
@@ -0,0 +1,45 @@
+pkgname = "collectd"
+pkgver = "5.12.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+        "--disable-pcie_errors",
+]
+configure_gen = []
+hostmakedepends = [
+        "automake",
+        "bison",
+        "flex",
+        "libtool",
+        "linux-headers",
+        "pkgconf",
+]
+makedepends = [
+        "hiredis-devel",
+        "i2c-tools-devel",
+        "jansson",
```

Comment:
```suggestion
        "jansson-devel",
```

Reply:
it's unable to find `sys/queues.h` with this change, do you know what package provides that?  I'm also not sure why that header gets lost, reading the template.py for jansson/jansson-devel.

Reply:
i don't think this change is the cause of that, since jansson-devel pulls jansson itself (since the -devel package pulls the main lib)

sys/queue.h is in `musl-bsd-headers` and nothing in this dep list pulls it already i think

Reply:
yep that's the right package, thanks!  I definitely bisected with just this change though, so I'm not sure why this caused it to not find the header, strange...